tests/regression/01-cpa/75-refine-def-exc.c

```diff
@@ -0,0 +1,16 @@
+//PARAM:
```

Comment:
This should be more explicit that we're testing def_exc and not intervals.
```suggestion
//PARAM: --enable ana.int.def_exc --disable ana.int.interval
```